theia-docker/Dockerfile

```diff
@@ -13,7 +13,8 @@ RUN yarn --pure-lockfile && \
     echo *.ts.map >> .yarnclean && \
     echo *.spec.* >> .yarnclean && \
     yarn autoclean --force && \
-    yarn cache clean
+    yarn cache clean ; \
+    yarn theia download:plugins
```

Comment:
you should call it before `yarn --production`.

Did me merge `@theia/cli` as a runtime dependency to some images already? we have to undo it